build/yamls/antrea-eks.yml

```diff
@@ -407,12 +407,16 @@ data:
     #   tls.crt: <TLS certificate>
     #   tls.key: <TLS private key>
     #selfSignedCert: true
+
+    # Enable Antrea NetworkPolicy CRDs. Setting it to true enables the controller
+    # watch for ClusterNetworkPolicy CRDs.
```

Comment:
just ClusterNetworkPolicies or the namespaced ones as well?

Reply:
Both. Let me update that.